libpay-2checkout/src/test/scala/com/wix/pay/twocheckout/TwocheckoutGatewayIT.scala

```diff
@@ -76,14 +78,37 @@ class TwocheckoutGatewayIT extends SpecWithJUnit with TwocheckoutTestSupport {
     }
 
     "fail with PaymentRejectedException for rejected transactions" in new Ctx {
```

Comment:
AFAIU this also checks that error will be successfully thrown if `orderNumber` is absent, right?

Reply:
Yep. This test was left as prove of old behaviour